src/main.c

```diff
@@ -0,0 +1,118 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include "../include/bmi.h"
+#include "../include/et.h"
```

Comment:
update include paths to `pet.h`

Reply:
Updated. But that main.c file might not be needed... Might delete...